internal/plan/plan.go

```diff
@@ -256,7 +263,7 @@ func ParseLayer(order int, label string, data []byte) (*Layer, error) {
 
 var fnameExp = regexp.MustCompile("^([0-9]{3})-([a-z](?:-?[a-z0-9]){2,}).yaml$")
 
-func ReadLayersDir(dirname string) ([]*Layer, error) {
+func ReadLayersDir(dirname string) (*Plan, error) {
```

Comment:
Shouldn't this still return []*Layers, after all it's literally "read layers dir"?  That's the original idea at least. The ReadDir below does return a *Plan, though.

Reply:
Good idea, fixed.